frontends/web/src/locales/en/app.json

```diff
@@ -827,6 +827,10 @@
       "close": "Close guide",
       "open": "Guide"
     },
+    "trackingModePortfolioChartMobile": {
+      "text": "Hold your finger on the chart and drag to see the value of your portfolio at different times in history.",
+      "title": "How to see historical values on the chart on mobile?"
```

Comment:
Sorry I proposed with "on mobile" as an intermediate solution, but as soon as it is in locize others might translate this temp. text. So I think it would be better if the entry just describes both desktop and mobile:

title: How to see historical values on the chart?
text: On desktop hover the cursor or on mobile touch and drag on the chart to see the value of your portfolio at different times in history.

Reply:
A simpler version which I think is more clear.

On desktop, hover the cursor over the chart. On mobile, hold your finger on the chart and drag horizontally.

Reply:
Just made the change. PTAL 🙏 

![Screen Shot 2022-11-09 at 03 31 06](https://user-images.githubusercontent.com/28676406/200722950-ca2108bf-ab75-4a06-857f-1af9032c63ba.png)